src/pages/ts-language-guide/golem-ts-sdk.mdx

```diff
@@ -9,239 +9,12 @@ import { MultiPlatformCommand } from "@/components/multi-platform-command"
   and these steps can be ignored.
 </Callout>
 
-If setting up the [Golem TypeScript SDK](https://github.com/golemcloud/golem-ts) manually, the following steps are required:
```

Comment:
As we are not documenting "circumventing" the app manifest elsewhere, I decided to remove this section